src/main/kotlin/com/cognifide/gradle/common/http/HttpClient.kt

```diff
@@ -75,20 +77,25 @@ open class HttpClient(private val common: CommonExtension) {
         common.prop.string("httpClient.basicPassword")?.let { set(it) }
     }
 
-    var basicCredentials: Pair<String?, String?>
-        get() = when {
-            basicUser.isPresent && basicPassword.isPresent -> (basicUser.get() to basicPassword.get())
-            else -> throw HttpException("HTTP client basic credentials are missing!")
-        }
+    var basicCredentials: Pair<String, String>?
```

Comment:
but here you have nullable, not empty string :D be consistent :)

Reply:
got it :)